include/bout/paralleltransform.hxx

```diff
@@ -156,6 +165,21 @@ public:
 
   bool canToFromFieldAligned() override { return true; }
 
+  bool twistShift(bool twist_shift_enabled, YDirectionType ytype) {
+    // Twist-shift only if field-aligned
+
+    if (ytype == YDirectionType::Aligned) {
+      // Twist-shift is required if field-aligned
+      if (not twist_shift_enabled) {
+        throw BoutException("'TwistShift = true' is required to communicate "
+            "field-aligned Field3Ds when using ShiftedMetric.");
+      }
+      return true;
+    } else {
+      return false;
+    }
+  }
```

Comment:
This is purely a style comment!

What do you think about the following?

```cpp
  bool twistShift(bool twist_shift_enabled, YDirectionType ytype) {
    // Twist-shift only if field-aligned
    if (ytype == YDirectionType::Aligned and not twist_shift_enabled) {
      throw BoutException("'TwistShift = true' is required to communicate "
                          "field-aligned Field3Ds when using ShiftedMetric.");
    }
    return ytype == YDirectionType::Aligned;
  }
```

Pros:
- Less indentation
- Logic is clearer

Cons:
- Conditional repeated

Otherwise, drop the `else` statement and just `return false`. If we always `return` (or `throw`) in the `if` branch, then we can remove the `else` statement as we must always be doing that if we get that far